packages/core/src/Prompt.ts

```diff
@@ -371,6 +371,7 @@ export function messagesToHistoryStr(messages: ChatMessage[]) {
 }
 
 export const defaultContextSystemPrompt = ({ context = "" }) => {
+  if (!context) return "";
```

Comment:
@parhammmm looks like a great idea, but why did you add it to this PR? did you run in a specific error condition? If so, can you send a new PR for this?

Reply:
@marcusschiesser cheers! I put it here since it was quite Gemini specific.`Context information is below.` would force Gemini to respond with `I cannot answer the question because the provided context does not mention ...` I can make a new PR if you still prefer it that way